src/View/Cell.php

```diff
@@ -183,9 +214,7 @@ public function render($template = null)
         };
 
         if ($cache) {
-            return $this->View->cache(function () use ($render) {
-                echo $render();
-            }, $cache);
+            return Cache::remember($cache['key'], $render, $cache['config']);
```

Comment:
Isn't this very similar to what View->cache() does? I guess I'm trying to avoid more cross package coupling points.